features/git-town-ship/current_branch/on_feature_branch/offline.feature

```diff
@@ -42,12 +42,11 @@ Feature: git town-ship: offline mode
       |         | git revert <%= sha 'feature done' %>           |
       |         | git checkout feature                           |
       | feature | git checkout main                              |
-      | main    | git checkout feature                           |
+      | main    | git reset --hard <%= sha 'Initial commit' %>   |
```

Comment:
These changes come from `git-ship` also running the sync steps. The changes here create the correct behavior of `git ship` in offline mode. They were not possible in the PR before. Looking back I could have avoided this by updating `git sync` first, then `git-ship`, since ship has a dependency on sync.